README.md

```diff
@@ -22,21 +22,25 @@ Some notable features include:
 * [Modeled configuration](#settings) that's automatically migrated through updates.
 * [Security](#security) as a top priority.
 
-## Contact us
+## Participate in the Community
 
-If you find a security issue, please [contact our security team](https://github.com/bottlerocket-os/bottlerocket/security/policy) rather than opening an issue.
+There are many ways to take part in the Bottlerocket community:
 
-If you're interested in contributing, thank you!
+- [Join on Meetup](https://www.meetup.com/bottlerocket-community/) to hear about the latest Bottlerocket (virtual/in-person) events and community meetings. The [next community meeting is August 24, 2022](https://www.meetup.com/bottlerocket-community/events/287425423/).
```

Comment:
Do we want to include specific dates here? That would mean we need to make sure to regularly update the README.

I think it may be better to just drop the last sentence, or state something about seeing the Meetup page for upcoming meeting details.

Reply:
How about a link to a COMMUNITY-MEETINGS.md file (containing a table of all past and planned community meetings?)

Or a GitHub Discussions thread made for announcing the community meetings?

Reply:
I like the idea of a Discussion thread. That increases the ways someone might discover our community meeting. And someone could follow that for any other updates we would want to post related to community meetings.

Reply:
Great question @stmcginnis. Typically, I wouldn't add short lived information to a readme, however this is a stop gap to advertise the current community meeting given that the repo is our only stable URL at the moment.

Long term, we have a website in-process and that's where this kind of information would go after it's established. 

Given that I'm going to be directing a lot of folks from upcoming events to this repo, I wanted to make sure there is a link directly to the next community meeting.

Reply:
Makes sense!